content/en/docs/gke/deploy/project-setup.md

```diff
@@ -66,6 +67,9 @@ Follow these steps to set up your GCP project:
   to learn how to check your quota and how to request an increase in quota.
   
 1. Initialize your project to ready it for Anthos Service Mesh installation.
+    ```
```

Comment:
Suggestion: code block formatting, use a common verb instead of "to ready"

```suggestion
1. Initialize your project to prepare it for Anthos Service Mesh installation:

    ```shell
```

Reply:
@joeliedtke PTAL at this PR🙏 @Bobgy

Reply:
I think we are preferring `bash` over `shell`

Reply:
@Bobgy Cool. Do you know if it significantly affects the formatting/rendering in Markdown? If it does, I'll try to make sure future PRs for non-GCP KF docs have `bash` 👍